sqlite/comdb2build.c

```diff
@@ -2311,6 +2312,13 @@ static char *gen_key_name(struct comdb2_ddl_context *ctx, const char *tabname,
     return keyname;
 }
 
+#define PK "PRIMARY_KEY"
```

Comment:
@akshatsikarwar thought you were suggesting to rename the proprocessor identifier "PK". How about prefixing with a $, "$PRIMARY_KEY"?